FPS.py

```diff
@@ -453,14 +463,9 @@ def EnrollStart(self,ID):
         self.SendCommand(packetbytes, 12)
         del packetbytes
         rp = self.GetResponse()
-        retval = 0
+        retval = "ACK"
         if not rp.ACK:
-            if rp.Error == rp.errors['NACK_DB_IS_FULL']:
-                retval = 1
-            elif rp.Error == rp.errors['NACK_INVALID_POS']:
-                retval = 2
-            elif rp.Error == rp.errors['NACK_IS_ALREADY_USED']:
-                retval = 3
+            retval = rp.Error
```

Comment:
If you return rp.Error value directly, you're not returning a valid integer value for another systems that uses this library. 
You can add a greater  value to retval that it corresponds to another greater value of rp.errors index constants, but you cannot broke the compatibility of another client systems